docs/user/dashboard/lens.asciidoc

```diff
@@ -156,11 +156,15 @@ A subset of *Lens* visualizations support value labels.
 [role="screenshot"]
 image::images/lens_value_labels_xychart_toggle.png[Lens Bar chart value labels menu]
 
+Note: it is not possible yet to control the labels position in bar visualizations.
+
 * *Pie*, *Donut*, and *Treemap*
 +
 [role="screenshot"]
 image::images/lens_value_labels_partition_toggle.png[Lens Pie chart value labels menu]
 
+Other type of visualizations do not support labels yet.
```

Comment:
I don't think we need this sentence. In the intro to the section, we already say `A subset of **Lens** visualizations support value labels.`